### PR TITLE
Implement feature #88

### PR DESCRIPTION
**Pull request by R0Wi**

### CODE REVIEW

**Comment by R0Wi** [comment]
@bahnwaerter coverage tracking in JS seems to be fixed again. Unfortunately Codecov doesn't report it correctly at the moment. If you look at the Github action it will show you a 100% coverage in JS. Also if you run the coverage locally it works fine

image

**Comment by R0Wi** [comment]
Build: https://github.com/R0Wi/workflow_ocr/suites/8023820543/artifacts/344560973

**Review by bahnwaerter — COMMENTED** [review]
This patch looks good to me, except for two minor problems regarding code format style. I reviewed all code changes but have not done any manual testing of this new feature on a fresh Nextcloud installation. The creator of the feature request has already tested the first working implementation of this patch and couldn't observe any errors. Therefore, feel free to release this new feature after solving the two minor code format style issues.

File: src/components/GlobalSettings.vue [hunk 1]
```diff
@@ -22,8 +22,7 @@
 
 <template>
 	<div>
-		<SettingsSection
-			title="Workflow OCR"
+		<SettingsSection title="Workflow OCR"
```
[the remark this answers is not in the document]
Comment (R0Wi) [reply src/components/GlobalSettings.vue:25]: Indeed VScode complains if it's not there, yes

File: tests/Unit/Service/OcrServiceTest.php [hunk 2]
```diff
@@ -52,9 +58,11 @@ public function setUp() : void {
 		$this->globalSettingsService = $this->createMock(IGlobalSettingsService::class);
 		$this->ocrProcessorFactory = $this->createMock(IOcrProcessorFactory::class);
 		$this->ocrProcessor = $this->createMock(IOcrProcessor::class);
+		$this->systemTagObjectMapper = $this->createMock(ISystemTagObjectMapper::class);
+		$this->logger = $this->createMock(LoggerInterface::class);
 		$this->fileInput = $this->createMock(File::class);
-
-		$this->ocrService = new OcrService($this->ocrProcessorFactory, $this->globalSettingsService);
+
```
[the remark this answers is not in the document]
Comment (R0Wi) [reply tests/Unit/Service/OcrServiceTest.php:64]: I'm a bit confused that the lint fix step did not fix this part 🙈

**Pull request merged**